dvc/utils/fs.py

```diff
@@ -134,3 +134,14 @@ def remove(path):
     except OSError as exc:
         if exc.errno != errno.ENOENT:
             raise
+
+
+def path_isin(child, parent):
+    """Check if given `child` path is inside `parent`."""
+
+    def normalize_path(path):
+        return os.path.normpath(fspath(path))
```

Comment:
It should be `fspath_py35()` instead of `fspath()` here, since in Python 3.6+ you may pass Path-like object to `os` utils directly.

Reply:
Changed, thanks.